tests/unit/engines/rl/inference/test_rl_inference_component.py

```diff
@@ -0,0 +1,73 @@
+# pylint: disable-all
+
+import pytest
+
+from recommender import User
+from recommender.engines.panel_id_to_services_number_mapping import PANEL_ID_TO_K
+from recommender.engines.rl.inference.rl_inference_component import RLInferenceComponent
+from recommender.models import Category
+
+
+@pytest.mark.parametrize("ver", ["v1", "v2"])
+def test_known_user(mongo, generate_users_and_services, mock_rl_pipeline_exec, ver):
+    K = PANEL_ID_TO_K[ver]
+    inference_component = RLInferenceComponent(K=K)
+
+    user = User.objects.first()
+    context = {"panel_id": ver, "search_data": {}, "user_id": user.id}
+
+    services_ids_1 = inference_component(context)
+
+    assert isinstance(services_ids_1, list)
+    assert len(services_ids_1) == PANEL_ID_TO_K.get(context["panel_id"])
+    assert all([isinstance(service_id, int) for service_id in services_ids_1])
+
+    services_ids_2 = inference_component(context)
+    assert services_ids_1 == services_ids_2
```

Comment:
Are we assuming here that if the same user requests recommendations using the same body request, the recommendations will be always the same?